tasks/main.yml

```diff
@@ -10,15 +8,15 @@
 
 - include_tasks: install.yml
   when: >
```

Comment:
@dverhelst Why not set the tags here when including the tasks, instead of tagging each task inside the imported file?

Reply:
Nice suggestion, overlooked tags all-together as I don't use them myself.
See 6efae44276ab61cf55ab5b92723366e5e9966ee5